game/src/components/heal.rs

```diff
@@ -0,0 +1,31 @@
+use bevy::prelude::*;
+
+///Pointer struct to mark components that need a Heal calculation
+#[derive(Component, Debug)]
+pub struct Heal {
+    pub heal_value: u32,
+}
+
+impl Heal {
+    pub fn new(heal_value: u32) -> Self {
+        Self { heal_value }
+    }
+}
+
+///Component that represents the healing factor
+#[derive(Component, Debug)]
+pub struct HealApplied {
```

Comment:
Same as `DamageApplied`, maybe something like `HealingFactor` would be better?

Reply:
completed